RFC/RFC-14-p2p-adapter.md

```diff
@@ -221,6 +229,8 @@ type NewPeerRoutingTableEvent = {
 
 Reference implementation:
 
+// TODO: Update this
```

Comment:
I don't think we need to update it right now, since the algorithm has the guarantees we need and it doesn't say anything regarding how the package will be encoded